guides/field-projection.md

```diff
@@ -0,0 +1,23 @@
+# Field Projection
+
+Some endpoints support field projection,
+which allows you to control which fields are returned in the response object.
+This is done with the use of fieldsets and projected fields.
+
+**Fieldsets** are predefined, named sets of fields.
+Fieldsets are provided by Wix for common use cases.
+For example, in the Contacts API,
+the `FULL` fieldset returns the full contact object,
+whereas the `BASIC` fieldset returns
+the contact's name, primary email, and primary phone number.
```

Comment:
Maybe add here the last two sentences that explain that there is always a default dataset, which is usually the full object - but look at the API's docs for details.   I think it's more relevant here where we explain what a fieldset is. 

I would keep the sentence about if "neither fieldsets nor projected fields are specified" where it is at the end.

Reply:
done